src/bundle/Resources/views/content/content_edit/content_create.html.twig

```diff
@@ -1,51 +1,13 @@
-{% extends 'EzPlatformAdminUiBundle:content/content_edit:content_edit_base.html.twig' %}
-
-{% trans_default_domain 'content_create' %}
-
-{% block details %}
-    <h2 class="text-muted">{{ 'creating_in_language'|trans({'%contentTypeName%': contentType.name, '%languageName%': language.name})|desc('Creating - %contentTypeName% in %languageName%') }}</h2>
-    <h1>
-        <svg class="ez-icon ez-icon-{{ contentType.identifier }}">
-            <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#{{ contentType.identifier }}"></use>
-        </svg>
-        {{ 'new_content_item'|trans({'%contentType%': contentType.name})|desc('New %contentType%') }}
-    </h1>
-
-    <div class="small">
-        {{ contentType.name }} / Parent Location ID: {{ parentLocation.id }}
-    </div>
-    {# @todo remove if statement once getDescription() bug is resolved in kernel #}
-    {% if contentType.descriptions is not empty %}
-        <div class="small text-muted">{{ contentType.description }}</div>
-    {% endif %}
-{% endblock %}
-
-{% block form_fields %}
-    {{ parent() }}
-    {{ form_widget(form.publish, {'attr': {'hidden': 'hidden'}}) }}
-    {{ form_widget(form.saveDraft, {'attr': {'hidden': 'hidden'}}) }}
-    {{ form_widget(form.cancel, {'attr': {'hidden': 'hidden'}}) }}
-{% endblock %}
-
-{% block right_sidebar %}
-    {% set content_create_sidebar_right = knp_menu_get('ezplatform_admin_ui.menu.content_create.sidebar_right', [], {'parentLocation': parentLocation, 'content_type': contentType}) %}
-    {{ knp_menu_render(content_create_sidebar_right, {'template': '@EzPlatformAdminUi/parts/menu/sidebar_right.html.twig'}) }}
-{% endblock %}
-
-{% block close_button %}
-    <a class="ez-content-edit-container__close" href="{{ path('_ezpublishLocation', {'locationId': parentLocation.id}) }}"></a>
-{% endblock %}
-
-{% block form_before %}
-    {{ ezplatform_admin_ui_component_group('content-create-form-before', {
-        'parentLocation': parentLocation,
-        'contentType': contentType
-    }) }}
-{% endblock %}
-
-{% block form_after %}
-    {{ ezplatform_admin_ui_component_group('content-create-form-after', {
-        'parentLocation': parentLocation,
-        'contentType': contentType
-    }) }}
-{% endblock %}
+{% embed 'EzPlatformAdminUiBundle:content/content_edit:content_edit_base.html.twig' %}
+    {% block content %}
```

Comment:
Can we provide more descriptive name for block content? Maybe `content_wrapper`?